lib/connect.js

```diff
@@ -128,7 +128,7 @@ const getTasks = (model, setupListeners) => {
     }
 
     const ctx = (error, opts) => {
-      options = opts;
+      options = { ...model.driverOptions, ...opts };
```

Comment:
To have a possibility to pass driver options to the connect function. In my case, I needed to reduce the value of `serverSelectionTimeoutMS` option to throw connection rejection error before test timeouts.